cfpb/design-stories
# Set up component testing via Vitest/Jest/Testing Library

Sets up a test runner and adds some tests for the `Button` component.

## Changes

- Updated Vite and ESLint configs
- Added button tests to `Button.test.tsx
- Updated docs in readme
- GH action workflow that runs tests on every PR

## How to test this PR

1. Pull down branch
1. `yarn && yarn test:ci` should run without errors
1. The [GH workflow](https://github.com/cfpb/design-stories/actions/runs/4748827028/jobs/8435441637) should pass

Closes #22.
